package.json

```diff
@@ -73,5 +73,6 @@
     "testMatch": [
       "<rootDir>/test/**/*.test.ts"
     ]
-  }
+  },
+  "packageManager": "pnpm@8.5.0"
```

Comment:
Allow installing pnpm via corepack, which is how I install it, and is the recommended way if you don't like `curl | sh`.